src/buildercore/lifecycle.py

```diff
@@ -114,7 +114,8 @@ def _last_ec2_start_time(stackname):
     nodes = find_ec2_instances(stackname, allow_empty=True)
 
     def _parse_datetime(value):
-        return datetime.strptime(value, "%Y-%m-%dT%H:%M:%S.%fZ")
+        assert value.tzname() == 'UTC', 'datetime object returned by the EC2 API is not UTC, needs timezone conversion'
+        return value.replace(tzinfo=None)
```

Comment:
Stripping timezone information results in a 'naive' datetime object, which is not good a state to be in. Your tests are also specifying `utc`, so it's strange.

Reply:
This was a quick fix that did the job of letting the code run, but it's open for improvement

Reply:
I'm certain my future self will find it, wrinkle his nose and feel compelled to fix it